src/loc.rs

```diff
@@ -239,3 +241,48 @@ impl Loc for JavaCode {}
 impl Loc for GoCode {}
 impl Loc for CssCode {}
 impl Loc for HtmlCode {}
+
+#[cfg(test)]
+mod tests {
+    use std::path::PathBuf;
+
+    use super::*;
+
+    #[test]
+    fn test_cloc_python() {
+        check_metrics!(
+            "\"\"\"Block comment\nBlock comment\n\"\"\"\n
+            # Line Comment\na = 42 # Line Comment\n",
+            "foo.py",
+            PythonParser,
+            loc,
+            [(cloc, 5, usize)]
```

Comment:
mmh... I see only 4 comments here, am I correct ?

Reply:
```
"""Block comment
Block comment
"""
#Line Comment
a = 42 # Line Comment
```

There are 5 lines because I put a `\n` after the second `Block Comment`